inductiva/geometry/cad_object.py

```diff
@@ -0,0 +1,236 @@
+
+        return self
+
+    def to_ground(self, inplace: bool = False):
+        """Translate the CADObject to the ground.
+
+        Args:
+            inplace: If True, the object is translated in place.
+        """
+
+        print("Translating object to the ground.")
+        data = self.data.translate([0., 0., -self.data.center[2]])
+
+        if inplace:
+            self.data = data
+
+        return self
+
+    def rotate_around_vector(self,
+                             vector: List[float],
+                             center: Optional[List[float]] = None,
+                             angle: float = 0.,
+                             inplace: bool = False):
+        """Rotate the CADObject by an angle around a vector.
+        
+        Args:
+            vector: 3D vector to rotate the object around.
+            center: 3D center point of the rotation.
+                Default is None, which is converted to [0., 0., 0.]
+            angle: Angle in degrees to rotate the object.
+            inplace: If True, the object rotation is saved in the data.
+        """
+
+        check_spatial_vector(vector, "Vector")
+        check_spatial_vector(center, "Center")
+
+        if center is None:
+            center = [0., 0., 0.]
+
+        print(f"Rotating around {vector} at the center point {center}"
+              f" by {angle} degrees")
+        data = self.data.rotate_vector(vector, angle, center, inplace)
+
+        if inplace:
+            self.data = data
+
+        return self
+
+    def rotate_around_axis(self,
+                           axis: Literal["x", "y", "z"],
+                           angle: float = 0.,
+                           inplace: bool = False):
+        """Rotate the CADObject by an angle around an cartesian axis.
+        
+        Args:
+            axis: Axis to rotate the object around.
+            angle: Angle in degrees to rotate the object.
+            inplace: If True, the object rotation is saved in the data.
+        """
+
+        axis_vector_dict = {
+            "x": [1., 0., 0.],
+            "y": [0., 1., 0.],
+            "z": [0., 0., 1.]
+        }
+
+        if axis in ["x", "y", "z"]:
+            vector = axis_vector_dict[axis]
+        else:
+            raise ValueError("Axis must be one of 'x', 'y', 'z'.")
+
+        print(f"Rotating around the {axis}-axis by {angle} degrees.")
+        data = self.data.rotate_vector(vector, angle, inplace=inplace)
+
+        if inplace:
+            self.data = data
+
+        return self
+
+    def scale(self, xyz: Union[float, List[float]], inplace: bool = False):
+        """Scale the CADObject by a vector.
+        
+        Args:
+            vector: float or 3D vector that define the scale factors,
+                overall and along x, y, and z, respectively. If a float
+                is passed the same uniform scale is applied to all axes.
+            inplace: If True, the object is scaled in place.
+        """
```

Comment:
Add a "Returns" section in the docstring to clarify the return type and behavior:
```
CADObject: The CADObject instance after scaling.
```